plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -949,6 +950,24 @@ protected WriteMapping legacyToWriteMapping(@SuppressWarnings("unused") Connecto
         throw new TrinoException(NOT_SUPPORTED, "Unsupported column type: " + type.getDisplayName());
     }
 
+    protected boolean preventTextualTypeAggregationPushdown(List<AggregateFunction> aggregates, List<List<ColumnHandle>> groupingSets)
```

Comment:
can this be static?

(that would help understand the state flow)

Reply:
Yes, should've been static from the start. thanks.